docs/examples/compare-to-eos.ipynb

```diff
@@ -0,0 +1,186 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9aaa5e0c-def6-4da5-9f63-b4c72a0de8f1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import orsopy\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import scipp as sc\n",
+    "import sciline\n",
+    "from ess import amor\n",
+    "from ess.reflectometry.types import *\n",
+    "from ess.amor.types import ChopperPhase"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "465f0c65-1305-4564-abe2-d70a0416a40d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "git clone https://github.com/jochenstahn/amor.git\n",
+    "wget -P raw https://public.esss.dk/groups/scipp/ess/amor/1/amor2023n000{608..614}.hdf"
```

Comment:
😱 this (and running the scripts below) looks like a big security red flag. I guess this is only meant to be temporary, but I think we should nevertheless avoid including such code/command execution in our repositories. We don't want to execute arbitrary code on our build servers or on our developer machines.

Reply:
I think this is a bit overdramatic. This is not "arbitrary code", it's a library maintained by a collaborator. Would you have the same concerns if Jochen had published his code on pypi and I had installed it using pip?

I could pin it to a particular commit hash. Then we don't have to worry about someone getting access to his repo and adding malicious code.

Reply:
I would have similar concerns if it was installed using `pip`. Not necessarily about malicious intent, but (1) people/projects can get hacked, and (2) some simply typos in a script could wipe someone's hard drive.

Reply:
Of course. Point is that we have other similar dependencies  maintained by few people who don't necessarily have strong security. This is not fundamentally different. What makes it a security red flag in this case?

Reply:
It is nothing particular to this case, it is always a problem, also in the other cases you have in mind (which ones?).

Reply:
I think orsopy could be counted in the same category. There was some talk about using a third party python package for corrections in nmx. I mean, it was not clear to me that here is were we draw the line. That's why I found your reaction surprising.

Would you be okay with this dependency if I pin it to a commit?

Reply:
Maybe you can clarify why this has to be a dependency (or otherwise installed by the package).

Reply:
The idea was to have a notebook for tracking agreement/disagreement with Jochens amor data reduction software.

I don't necessarily think this has to be part of the docs page. But it would be convenient if it was.

Reply:
Can we just save that notebook somewhere (so we have it for reproducibility), but not execute it in our builds? Instead, save the outputs from jochen's code and make them available in pooch, and make a different notebook that just compares the results?

Reply:
Good idea. I'll do that.